### ISSUE TITLE
Order by desc/asc

### ISSUE THREAD

**Issue by jcubic** [cloudson/gitql#50]
It seems that the two queries return the same results:
```
git ql "select message from commits order by date asc limit 10"
git ql "select message from commits order by date desc limit 10"
```

**Comment by jsixface** [comment]
Looks like if the sort column is not present in the selected column list, sorting does not work.
The above queries with the `date` in the select column seem to work
```
git ql "select message, date from commits order by date asc limit 10"
git ql "select message, date from commits order by date desc limit 10"
```

**Comment by luizperes** [comment]
Nice to point that out @jsixface! I believe that the current code is ignoring the `date` in the first example. Perhaps just a matter of that to the symbol table. I will try to check that out later. Thanks.

**Comment by luizperes** [comment]
Hi @jcubic, that was fixed on #52 by @jsixface, thanks!

**Comment by jcubic** [comment]
Oh, I've completly forget about this project I've just installed it on my current Linux Box.
